guides/common/attributes-katello.adoc

```diff
@@ -4,4 +4,5 @@
 :installer-scenario: foreman-installer --scenario katello
 :installer-log-file: /var/log/foreman-installer/katello.log
 :smartproxy_port: 9090
-:dnf-modules: katello:el8 pulpcore:el8
+:dnf-module: katello:el8
```

Comment:
Naming suggestion: `main-dnf-module`? I am unsure if this makes things easier. Downside: sort in alphabetical order and the attributes are no longer side by side.

IMHO it's fine as is.